### ISSUE TITLE
Unable to use ui.message in another thread

### ISSUE THREAD

**Issue by CyrilleB79** [mltony/nvda-bluetooth-audio#16]
### Issue
Consider the following code to paste in NVDA's console
```
import threading
import ui
def test():
 threading.Thread(target=thread).start()

def thread():
 ui.message("Hello world")
```

Then, in the console, type:
`test()`

### Actual result
With this add-on, an error is logged and the message is not spoken.

### Expected result
`ui.message` should be usable from any thread as when this add-on is not installed in NVDA.
